src/ducks/Studio/Compare/index.js

```diff
@@ -24,8 +24,15 @@ const Compare = ({
 
   useEffect(
     () => {
+      const mixedProjects = [
+        ...allProjects,
+        { slug: 's-and-p-500', name: 'S&P500', ticker: 'SPX' },
+        { slug: 'gold', name: 'Gold', ticker: 'Gold' }
+      ]
       setProjects(
-        allProjects.filter(project => project.slug !== slug).sort(projectSorter)
+        mixedProjects
+          .filter(project => project.slug !== slug)
+          .sort(projectSorter)
```

Comment:
`SPX` and `Gold` could be moved outside of the function as a constant array, e.g. `const ADDITIONAL_PROJECTS = [{}, {}, ...]`, and appended to the `allProjects` via `allProjects.concat(ADDITIONAL_PROJECTS)`. This will give us a benefit in speed since no new objects will be created on function call, and `concat` is slightly faster for array concatenations

Reply:
Thx. concat looks better.